lib/ide/src/graph_api.rs

```diff
@@ -0,0 +1,210 @@
+//! TODO TODO
```

Comment:
I created double_representation module. We should call and agree some module structure.